secondRepository.html

```diff
@@ -0,0 +1,35 @@
+<! DOCTYPE html>
+<html>
+<head>
+    <title> second branch</title>
+    <link rel="stylesheet" href="markup.css">
+
+</head>
+<body>
+<h1> Creating another branch in a repository</h1>
+
+
+<p> Creating a nother branch as an example for collaboration working style via  <code>checkout -b secondBranch</code> command</p>
+<img src="src/001.png" alt="Creating a repo via GitHub" width="600"><br><br>
+
+<p> After adding all the necessary update and changes in this branch we need to create a MR (Merge request, PS:in some company they may call as PR for pull request words) </p>
+<img src="src/001.png" alt="Creating a repo via GitHub" width="600"><br><br>
+
+<p> Your colleagues who appointed as a reviewer may put some commands or they may mention issues for your updates like below</p>
+<img src="src/001.png" alt="Creating a repo via GitHub" width="600"><br><br>
+<p> After changes for the improvement of these issues needs to push all changes into the branch again and asking review again. </p>
+
+
+<p> After colleagues acceptance, merge button turn to green. Then you can merge all the changes from secondBranch to the master branch</p>
+<img src="src/001.png" alt="Creating a repo via GitHub" width="600"><br><br>
+
+
+<p> After mege process we can delete the secondBranch with <code>git -d secondBranch</code> command</p>
+<img src="src/001.png" alt="Creating a repo via GitHub" width="600"><br><br>
+
+<p> I am a hidden bug. Developer accidentally forgot me in here.</p>
```

Comment:
A note from a QA tester colleague: There is a bug. Could you sort this issue, please